### PR TITLE
feat: doc refresh

### PR DESCRIPTION
**Pull request by davemooreuws**

### CODE REVIEW

File: src/components/GettingStarted.tsx [hunk 1]
```diff
@@ -1,31 +1,94 @@
...
+      'Learn about the Nitric architecture and how it works under the hood.',
+  },
+  {
+    href: '/getting-started/quickstart',
+    name: 'Get started with Nitric',
+    icon: RocketLaunchIcon,
+    description: 'Start deploying in 15-minutes with our Quick Start guide.',
+  },
+  {
+    href: '/concepts/extending-nitric',
```
Comment (elisselockhart) [inline src/components/GettingStarted.tsx:77]: For now, can we link this to /reference/providers/custom/building-custom-provider? We can come back later and create new content for the concepts section as part of another story.

File: src/components/GettingStarted.tsx [hunk 2]
```diff
@@ -1,31 +1,94 @@
...
+  )
+}
+
+const gettingStartedBlocks = [
+  {
+    href: '/concepts/introduction',
+    name: 'What is Nitric',
+    icon: InformationCircleIcon,
+    description:
+      'Learn about the Nitric architecture and how it works under the hood.',
```
Comment (elisselockhart) [inline src/components/GettingStarted.tsx:68]:
```suggestion
      'Learn about the Nitric architecture and how the framework works under the hood.',
```

File: src/components/GettingStarted.tsx [hunk 3]
```diff
@@ -1,31 +1,94 @@
...
+  },
+  {
+    href: '/getting-started/quickstart',
+    name: 'Get started with Nitric',
+    icon: RocketLaunchIcon,
+    description: 'Start deploying in 15-minutes with our Quick Start guide.',
```
Comment (elisselockhart) [inline src/components/GettingStarted.tsx:74]:
```suggestion
    description: 'Start building and deploy in minutes with our Quick Start guide.',
```

File: src/components/Concepts.tsx [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+import { cn } from '@/lib/utils'
+import { ResourcePattern } from './ResourcePattern'
+
+const items = [
+  {
+    href: '/concepts/how-nitric-works',
+    name: 'How Nitric works',
+    description:
+      'The help forum is a great place to get help with any questions about Nitric.',
```
Comment (elisselockhart) [inline src/components/Concepts.tsx:18]:
```suggestion
      'Learn about the components of Nitric and how it works for development and deployment.',
```

File: src/components/Concepts.tsx [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+      squares: [
+        [0, 1],
+        [1, 3],
+      ],
+    },
+  },
+  {
+    href: '/concepts/comparison',
+    name: 'Comparison with other tools',
+    description: 'How Nitric differs from alternative solutions.',
```
Comment (elisselockhart) [inline src/components/Concepts.tsx:31]:
```suggestion
    description: 'See how Nitric differs and fits alongside other solutions.',
```

**Review by elisselockhart — DISMISSED** [review]

**Comment by HomelessDinosaur** [comment]
I think it looks great and love the navigation changes. Two small things, 
1. I'm not sure about the difference in font size between the `Concepts` section and the `Libraries` section. Would like to hear someone else's opinion on that.
Screenshot 2024-05-14 at 2 12 13 pm

2. When you click the individual cloud provider link on the sidebar it takes you to the whole nav (and starts at the bottom)
Sidebar

Screenshot 2024-05-14 at 2 18 11 pm

This is different to the behaviour with all the other links, where the sidebar changes to just the information about that specific link. Currently no matter which cloud provider is clicked it shows the above sidebar. Suggesting that the behaviour looks more like this with "More Providers".

Screenshot 2024-05-14 at 2 21 44 pm

**Review by HomelessDinosaur — APPROVED** [review]

**Review by elisselockhart — APPROVED** [review]

**Pull request merged**